docs/programatically-triggering-polling.adoc

```diff
@@ -1,59 +0,0 @@
-= Programatically Triggering Polling
-
-Aktualizr normally runs as a systemd service and regularly polls for updates. The default polling interval is set at 10s, and this should be increased by a few orders of magnitude in a production system.
-
-It is also possible to trigger the update cycle manually. This is done by stopping the systemd service and running aktualizr with the `--running-mode=once` option or by setting `running_mode = once` in the `[uptane]` configuration section.
-
-When disabling polling via a configuration file fragment, be warned that the default systemd service file shipped with Aktualizr will attempt to automatically restart the service when it exits.
-
-Please note that the following interface may change significantly in the future, possibly to a socket or D-BUS API.
-
-To run the update cycle manually, first stop and disable the aktualizr background service:
-
-    systemctl stop aktualizr && systemctl disable aktualizr
-
-Assuming aktualizr is being built using Yocto and meta-updater, the systemd service can also be disabled by default by adding the following to `local.conf`:
-
-    SYSTEMD_AUTO_ENABLE_aktualizr = "disable"
-
-With the background service disabled, trigger an update cycle by running the following command:
-
-    aktualizr --running-mode=once
-
-
-== Separately running individual update stages
-
-It is sometimes necessary to download an update when there is network connectivity then perform the installation later, without relying on the network. Note that aktualizr is always _safe_ against network drops. The worst that can happen if the network becomes unavailable is that the update will fail and need to be retried later.
-
-To use this mode, follow the same instruction above to disable the aktualizr systemd service, then instead of running `aktualizr --running-mode=once`, do the following:
-
-Fetch the Uptane metadata:
-
-    aktualizr --running-mode=check
-
-Download an update:
-
-    aktualizr --running-mode=download
-
-After this point the installation can proceed without network connectivity. If you are using a hardware device, unplug the cable. Under Qemu (for example using the  `meta-updater/scripts/run-qemu-ota --no-gui` script), the following commands on the Qemu console can be used to disable the network:
-
-    (qemu) info network
-    hub 0
-    \ hub0port1: e1000.0: index=0,type=nic,model=e1000,macaddr=ca:fe:6a:ee:98:fa
-    \ hub0port0: user.0: index=0,type=user,net=10.0.2.0,restrict=off
-    (qemu) set_link user.0 off
-    (qemu) set_link e1000.0 off
```

Comment:
This qemu bit might just be for testing and debugging, but might be worth keeping around even if just for that purpose. Could we still keep it as an appendix or something?

Reply:
I'm pretty reluctant to just scatter qemu documentation around inside . I suppose there's no great harm in leaving it; we'd just tag it and ignore it when we include in the ats garage docs...but still, it doesn't feel right. All it tells you is "this is how you can make sure it's working", which is only really useful if you're an aktualizr developer working on the feature.

Reply:
Fair point, most users won't care. Does it make sense to have a separate place to put things like that? Alternatively, we could just integrate it directly into an oe-selftest case. Some of those test cases were basically translated directly from existing documentation, too.